macros/form.njk

```diff
@@ -123,7 +123,7 @@
 
 {% macro submitButton(label = 'Continue', id = '') %}
   <div class="form-group">
-    <input type="submit" class="button" value="{{ t(label) }}" {% if id %}id="{{ id }}"{% endif %}>
+    <input type="submit" class="button" value="{{ t(label) }}"{% if id %}name="{{ id }}"{% endif %} {% if id %}id="{{ id }}"{% endif %}>
```

Comment:
why is this needed?

Reply:
I can't remember now as it was before xmas.. I'll remove it and see what happens

Reply:
have added a link to the prototype so you can see what the end result looks like

Reply:
Have remembered. Need the name on the button so that when it posts back it is part of the form object so I can check which button has been clicked